packages/lending/src/tokens/gnosis.json

```diff
@@ -0,0 +1,37 @@
+{
+  "XDAI": {
+    "chainId": 100,
+    "address": "0x0000000000000000000000000000000000000000",
+    "decimals": 18,
+    "symbol": "XDAI",
+    "name": "xDAI"
+  },
+  "WXDAI": {
+    "chainId": 100,
+    "address": "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d",
+    "decimals": 18,
+    "symbol": "WXDAI",
+    "name": "Wrapped XDAI"
+  },
+  "USDC": {
+    "chainId": 100,
+    "address": "0xDDAfbb505ad214D7b80b1f830fcCc89B60fb7A83",
+    "decimals": 6,
+    "symbol": "USDC",
+    "name": "USD//C on xDAI"
+  },
+  "USDT": {
+    "chainId": 100,
+    "address": "0x4ECaBa5870353805a9F068101A40E0f32ed605C6",
+    "decimals": 6,
+    "symbol": "USDT",
+    "name": "Tether USD on xDai"
+  },
```

Comment:
Remove USDT as there is no usage of `gnosisTokens.USDT`

Reply:
removed